src/RA_Dlg_MemBookmark.cpp

```diff
@@ -873,10 +874,10 @@ void Dlg_MemBookmark::OnLoad_NewRom()
 
 void Dlg_MemBookmark::GenerateResizes(HWND hDlg)
 {
-    RARect windowRect;
-    GetWindowRect(hDlg, &windowRect);
-    pDlgMemBookmarkMin.x = windowRect.Width();
-    pDlgMemBookmarkMin.y = windowRect.Height();
+    auto windowRect{ std::make_unique<RARect>() };
+    GetWindowRect(hDlg, LPRECT{ *windowRect });
+    pDlgMemBookmarkMin.x = windowRect->Width();
+    pDlgMemBookmarkMin.y = windowRect->Height();
 
     vDlgMemBookmarkResize.push_back(ResizeContent(hDlg,
         GetDlgItem(hDlg, IDC_RA_LBX_ADDRESSES), ResizeContent::ALIGN_BOTTOM_RIGHT, TRUE));
```

Comment:
I tested by using a managed pointer and it worked.
`RARect` and `RECT` were 16 bytes.
`RARect*` and `LPRECT` were 4 bytes.